CRM/Case/BAO/Case.php

```diff
@@ -2134,8 +2130,8 @@ public static function mergeCases(
       return $mainCaseIds;
     }
 
-    $activityTypes = CRM_Core_PseudoConstant::activityType(TRUE, TRUE, FALSE, 'name');
-    $activityStatuses = CRM_Core_PseudoConstant::activityStatus('name');
+    $activityTypes = CRM_Activity_BAO_Activity::buildOptions('activity_type_id', 'validate');
+    $activityStatuses = CRM_Activity_BAO_Activity::buildOptions('activity_status_id', 'validate');
```

Comment:
this is only used once to get         'status_id' => array_search('Completed', $activityStatuses),  - we could just do a getKey call

Reply:
Yes, done